Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -42,7 +42,7 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
             client.apiClient
         )
     
-    private let requestedChannelsLimit = 25
+    private let requestedChannelsLimit: Int
```

Comment:
Do we even need this?
```suggestion
```

We can then later just use:
`let limit = limit ?? query.pagination.pageSize` instead of `let limit = limit ?? requestedChannelsLimit`